diff --git a/dist/http/express-loader.js b/dist/http/express-loader.js
index 149a3f33962e1fe79a1457e54ad4d1333a7156ef..1f6c900384926ff4e24eed65790e149a26ceaad4 100644
--- a/dist/http/express-loader.js
+++ b/dist/http/express-loader.js
@@ -65,7 +65,7 @@ async function expressLoader({ app, container, }) {
             prefix: `${configModule?.projectConfig?.redisPrefix ?? ""}sess:`,
         });
     }
-    app.set("trust proxy", 1);
+    app.set("trust proxy", false);
     /**
      * Method to skip logging HTTP requests. We skip in test environment
      * and also exclude files served by vite during development
